apps/src/lib/client/tx.rs

```diff
@@ -1140,28 +1142,12 @@ pub async fn broadcast_tx(
         _ => panic!("Cannot broadcast a dry-run transaction"),
     };
 
-    let websocket_timeout =
-        if let Ok(val) = env::var(ENV_VAR_ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT) {
-            if let Ok(timeout) = val.parse::<u64>() {
-                Duration::new(timeout, 0)
-            } else {
-                Duration::new(300, 0)
-            }
-        } else {
-            Duration::new(300, 0)
-        };
-
-    let mut wrapper_tx_subscription = TendermintWebsocketClient::open(
-        WebSocketAddress::try_from(address.clone())?,
-        Some(websocket_timeout),
-    )?;
+    let rpc_cli = HttpClient::new(address)?;
 
-    let response = wrapper_tx_subscription
-        .broadcast_tx_sync(tx.to_bytes().into())
-        .await
-        .map_err(|err| WsError::Response(format!("{:?}", err)))?;
+    // TODO: timeout?
```

Comment:
yes, one that can be set with an env var. Needed for the ci

Reply:
we need to hack away at tendermint-rs once more then. their `HttpClient` uses a hardcoded 30 second timeout iirc.

Reply:
Ah, as long as there is a timeout, we should be fine. We just can't let it run forever.